packages/p2p/src/components/verification/verification.tsx

```diff
@@ -23,10 +21,10 @@ const VerificationWrapper = ({ should_wrap, children }) => {
         );
     }
 
-    return children;
+    return children as JSX.Element;
```

Comment:
```suggestion
    return <>{children}</>;
```

Instead of giving type to children which makes no use of `PropsWithChildren`, it should work too

Reply:
yes. that would work